### PR TITLE
kde-misc/kolor-manager: Frameworks merged to master

### PR DESCRIPTION
**Pull request by a17r**

### CODE REVIEW

File: media-gfx/synnefo/synnefo-9999.ebuild [hunk 1]
```diff
@@ -4,45 +4,29 @@
...
-	SRC_URI="https://github.com/oyranos-cms/Synnefo/archive/${PV}.tar.gz -> ${P}.tar.gz"
-	KEYWORDS="~amd64 ~x86"
-fi
+[[ ${PV} == 9999 ]] || \
+SRC_URI="https://github.com/oyranos-cms/Synnefo/archive/${PV}.tar.gz -> ${P}.tar.gz"
 
 LICENSE="BSD-2"
 SLOT="0"
-IUSE="qt5"
+[[ ${PV} == 9999 ]] || \
```
Comment (johu) [inline media-gfx/synnefo/synnefo-9999.ebuild]: Is this line so long that it needs a line break?

Comment (a17r) [inline media-gfx/synnefo/synnefo-9999.ebuild]: Not really, but I remember ekeyword not working with KEYWORDS on the same line?

Comment (johu) [inline media-gfx/synnefo/synnefo-9999.ebuild]: Works for me. Please double check and if it works for you change it to a one liner.

**Comment by johu** [comment]
Thanks. Picked 15e0cf1d4d4d79ca33be2f35d4006a19c44d3faf and e78428586689a9d863a60353b2990f9967e595e5.

**Pull request closed without merging**